lmdb/cpython.c

```diff
@@ -2196,13 +2196,19 @@ cursor_get_multi(CursorObject *self, PyObject *args, PyObject *kwds)
                 } else {
                     /* dupfixed, MDB_GET_MULTIPLE returns batch, iterate values */
                     int len = (int) self->val.mv_size/arg.dupfixed_bytes; // size_t?
-                    for(i=0; i<len; i++){
-                        // TODO Handle as_buffer?
-                        val = PyBytes_FromStringAndSize(
-                            (char *) self->val.mv_data+(i*arg.dupfixed_bytes),
-                            (Py_ssize_t) arg.dupfixed_bytes);
+                    for(i=0; i<len; i++) {
+                        char *val_data = self->val.mv_data + (i * arg.dupfixed_bytes);
```

Comment:
You need a (char *) on the right side of the equals. mv_data field is  void *, so can't be added to.